src/components/popups/popups.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+angular.module('openshiftCommonUI')
+  // This triggers when an element has either a toggle or data-toggle attribute set on it
+  .directive('toggle', function(IS_IOS) {
```

Comment:
The IS_IOS constant be added to the CommonUI module rather than depending on the applications to set it.